src/main/java/org/openstreetmap/atlas/geography/atlas/command/AtlasGeoJSONSubCommand.java

```diff
@@ -61,7 +61,7 @@ protected int finish(final CommandMap command)
     @Override
     protected void handle(final Atlas atlas, final CommandMap command)
     {
-        getWriter(command).write(atlas.asGeoJson(x -> true).jsonObject());
+        getWriter(command).write(atlas.asGeoJson(x -> true));
```

Comment:
Suggestion - provide an overloaded asGeoJson() that that in turn calls asGeoJson(x -> true)